app/cdap/components/PipelineTriggers/PipelineListCompositeTab/index.tsx

```diff
@@ -338,16 +355,20 @@ const PipelineListCompositeTabView = ({
           >
             {T.translate(`${PREFIX}.configComputeProfie`)}
           </PipelineTriggerComputeProfileButton>
-          <EnableGroupTriggerButton
-            color="primary"
-            disabled={state.isNameInvalid || triggersGroupToAdd.length === 0}
-            variant="contained"
-            onClick={() => addGroupTriggerClick()}
-            data-cy="enable-group-trigger-btn"
-            data-testid="enable-group-trigger-btn"
-          >
-            {T.translate(`${PREFIX}.addNewTrigger`)}
-          </EnableGroupTriggerButton>
+          <CustomTooltip arrow title={state.triggerNameError || emptyTriggerErrorMsg}>
```

Comment:
Are you sure this should use a tooltip? usually we try not to use tooltips for stuff that that has intractability?

Reply:
learned a new word today

Reply:
what's the advice here? I've set the error message in the trigger name input, but the configuration page could be too long if there are many candidate pipelines and users will ignore it. We need a proper way to guide users why the enable button is disabled

Reply:
use richtooltip

Reply:
i used it for a similar thing in the sidepanel

Reply:
its not that a popup menu is bad per se - its using the `tooltip` component to accomplish it.

Reply:
@sumengwang it should've been https://en.wiktionary.org/wiki/interactability but the dang autocorrect don't like me

Reply:
nvm i read this wrong - i thought the button was part of the tooltip content 😞